### DISCUSSION TITLE
Mengatasi Warning: Error in tagAssert: Expected an object with class 'shiny.tag'.

### DISCUSSION THREAD

**Discussion by ThesionMS** [teamalgoritma/community#679] [Q&A] [answered]
![image](https://github.com/teamalgoritma/community/assets/108519903/0fd7c1c3-e54e-4e17-8fb9-95168540d1c1)
![image](https://github.com/teamalgoritma/community/assets/108519903/d3f6d1f3-48f3-455c-bffc-8c52e18aaa7b)

Saya ingin menambahkan 3 chart, tapi tampilan eror seperti itu

**Comment by ramdhinafinita** [comment, marked as answer]
Dear @ThesionMS 
Output yang ingin dikeluarkan jika berupa plot seharusnya renderplotly bukan renderinfobox. Bisa dicek terlebih dulu di cheatsheet shinynya: https://raw.githubusercontent.com/rstudio/cheatsheets/main/shiny.pdf
